src/component/misc/ConfirmCancelDialog.tsx

```diff
@@ -42,7 +44,7 @@ const ConfirmCancelDialog: React.FC<ConfirmCancelDialogProps> = (props) => {
         <ButtonToolbar className="float-right">
           <Button
             id={`${props.id}-submit`}
-            color="primary"
+            color={props.confirmColor || "primary"}
```

Comment:
It would make sense to put the default value into the function component signature rather than `||` it here.